build/aliases.yaml

```diff
@@ -3,6 +3,9 @@ css:
   - 'postcss'
   - 'cssmin'
 
+svg:
```

Comment:
how is this to be used? what's the difference between using this and simply typing `svgo [...]`?